Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -205,6 +205,9 @@ std::string CVolumeWiiCrypted::GetInternalName() const
 std::map<IVolume::ELanguage, std::string> CVolumeWiiCrypted::GetNames() const
 {
 	std::unique_ptr<IFileSystem> file_system(CreateFileSystem(this));
+	if (file_system == nullptr)
+		return std::map<IVolume::ELanguage, std::string>();
```

Comment:
You can just do `return {};` to return an empty map. Also you can just use `!` to check for nullity of the unique_ptrs.